### ISSUE TITLE
Backlinks mention duplicate links

### ISSUE THREAD

**Issue by zubayrrr** [hikerpig/gatsby-project-kb#12]
The backlinks reference shows duplicates of a mentioned link(in the note), shouldn't backlinks mention the link only once regardless of how many times it was mentioned in the note?

![image](https://user-images.githubusercontent.com/31969517/119281229-846ce000-bc52-11eb-88f5-5281398a08f3.png)

Thank you again for this great theme, I'll try to fix these issues myself but I am still a noob.

**Comment by zubayrrr** [comment]
Okay, I didn't realize this is a feature, not a flaw. 😅

**Comment by hikerpig** [comment]
I think it'd be better if the actual refWord  can be highlighted so it won't be confusing at first glance like this.
